templates/hooks/layerversion/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,5 @@
+err = customPreDelete(r,rm,ctx)
+if err != nil{
+    return nil, err
+}
```

Comment:
Golang supports putting instantiations and conditionals into one-liners
```suggestion
if err = customPreDelete(r,rm,ctx); err != nil {
    return nil, err
}
```

Reply:
Done.